src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        final int MAX_TASK = 100;
+        Task[] tasks = new Task[MAX_TASK];
```

Comment:
`tasks` here is a global variable... Is there a reason you have to pass them explicitly into your methods such as `unmarkTask`?